src/main/resources/build.template.properties

```diff
@@ -9,6 +9,11 @@
 # For staging server, this must match the ID of the app your created on GAE.
 app.id = teammates-john
 
+# This is the deployment region of the app.
+# For dev server, this could be anything.
+# For staging/production server, this must match the region of your GAE application.
+app.region=us-central1
```

Comment:
@damithc I forgot to bring your attention to changes needed in the `src/main/resources/build.properties` file with the release of V7.10.0 which might be the reason for errors in #10974

Reply:
I see. How do I find the region used for the production version.

Reply:
@damithc you can type `gcloud app describe --project <project-name>` (replace `<project-name` with the project in GAE)

You should see something like this:
```
authDomain: gmail.com
codeBucket: staging.myapp-1337.appspot.com
defaultBucket: myapp-1337.appspot.com
defaultHostname: myapp-1337.appspot.com
featureSettings:
  splitHealthChecks: true
gcrDomain: us.gcr.io
id: myapp-1337
locationId: us-central
name: apps/myapp-1337
servingStatus: SERVING
```

For teammates, we should be `us-central` - so region should be `us-central1` (the docs mention that for this region we have to map the name slightly differently):
```
Note: Two locations, which are called europe-west and us-central in App Engine commands and in the Google Cloud Console, are called europe-west1 and us-central1, respectively, elsewhere in Google documentation.
```

Reply:
Got it. Thanks both. Will update and deploy again soon.